src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java

```diff
@@ -699,6 +683,28 @@ private void scrape(List<MetricFamilySamples> mfs) {
       samples.add(new MetricFamilySamples.Sample(
           "cloudwatch_exporter_scrape_error", new ArrayList<>(), new ArrayList<>(), error));
       mfs.add(new MetricFamilySamples("cloudwatch_exporter_scrape_error", Type.GAUGE, "Non-zero if this scrape failed.", samples));
+
+      /**
+       * New static cloudwatch_exporter_build_info implementation. Sets a gauge to 1 and creates two label/value
+       * pairs that can be displayed (Grafana style UI).
+       * NOTE - When releasing a new version the private class variables must be set. (lines 31 and 32).
```

Comment:
This is not great, I *will* forget to do that. Maven knows the version when building, is there any way that can be passed in during the build?

Reply:
Work in progress, but i'm not very familiar with maven. In the  pom.xml file that is used to build the project do you manually set the new version in the <version> tag or is it done dynamically?